MykSuite/src/main/java/com/infomaniak/core/myksuite/ui/screens/components/AppStorageQuotas.kt

```diff
@@ -38,12 +38,12 @@ import com.infomaniak.core.myksuite.ui.theme.Typography
 @Composable
 internal fun AppStorageQuotas(modifier: Modifier) {
     Column(modifier = modifier, verticalArrangement = Arrangement.spacedBy(Margin.Medium)) {
-        KSuiteApp.entries.forEach { AppStorageQuota(app = it) }
+        KSuiteAppWithQuotas.entries.forEach { AppStorageQuota(app = it) }
```

Comment:
Maybe we could use "product" over "app" in the enum name and the AppStorageQuota's argument name to better tell it apart from the KSuiteApp?

Reply:
You're right, product is a good name.  Even the "AppStorageQuotas" should be renamed with `product`

I'll do this in the next PR because I already modified a lot this part and changing it here would create conflict

Reply:
In Fact I think i'll do it in a separate PR, it will be the 12th